From 8ab6016b540acf03b627277b363025a5c66940ba Mon Sep 17 00:00:00 2001
From: Dave Kover <kover@cyngn.com>
Date: Mon, 22 Feb 2016 14:33:31 -0800
Subject: [PATCH 18/21] Messaging: Switch to mipmap launcher icons

Insert the launcher icon as mipmap. Change references.

This commit squashes the following commits from CM 13.0:
b6065457580199c897f0970364608540a90c4ef3 Add new colors and icon.
33e931646d82761758478375d13b27d9cd0c3302 Bring back original color

Change-Id: Ie7ebc19d71b46a76b4f48f9e5fec1b76ec6dbe5f
Ticket: MMS-216
---
 AndroidManifest.xml                                 |   2 +-
 res/layout/widget_conversation.xml                  |   2 +-
 res/{drawable-hdpi => mipmap-hdpi}/ic_launcher.png  | Bin
 res/{drawable-mdpi => mipmap-mdpi}/ic_launcher.png  | Bin
 .../ic_launcher.png                                 | Bin
 .../ic_launcher.png                                 | Bin
 .../ic_launcher.png                                 | Bin
 7 files changed, 2 insertions(+), 2 deletions(-)
 rename res/{drawable-hdpi => mipmap-hdpi}/ic_launcher.png (100%)
 rename res/{drawable-mdpi => mipmap-mdpi}/ic_launcher.png (100%)
 rename res/{drawable-xhdpi => mipmap-xhdpi}/ic_launcher.png (100%)
 rename res/{drawable-xxhdpi => mipmap-xxhdpi}/ic_launcher.png (100%)
 rename res/{drawable-xxxhdpi => mipmap-xxxhdpi}/ic_launcher.png (100%)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index bb822fb..94fefeb 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -58,7 +58,7 @@
     <application
         android:name="com.android.messaging.BugleApplication"
         android:allowBackup="false"
-        android:icon="@drawable/ic_launcher"
+        android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:theme="@style/BugleTheme"
         android:supportsRtl="true">
diff --git a/res/layout/widget_conversation.xml b/res/layout/widget_conversation.xml
index a9cda98..eaf1dcf 100644
--- a/res/layout/widget_conversation.xml
+++ b/res/layout/widget_conversation.xml
@@ -39,7 +39,7 @@
             android:freezesText="true" />
         <ImageView
             android:id="@+id/launcher_icon"
-            android:src="@drawable/ic_launcher"
+            android:src="@mipmap/ic_launcher"
             android:layout_height="wrap_content"
             android:layout_width="0dip"
             android:layout_weight = "1"
diff --git a/res/drawable-hdpi/ic_launcher.png b/res/mipmap-hdpi/ic_launcher.png
similarity index 100%
rename from res/drawable-hdpi/ic_launcher.png
rename to res/mipmap-hdpi/ic_launcher.png
diff --git a/res/drawable-mdpi/ic_launcher.png b/res/mipmap-mdpi/ic_launcher.png
similarity index 100%
rename from res/drawable-mdpi/ic_launcher.png
rename to res/mipmap-mdpi/ic_launcher.png
diff --git a/res/drawable-xhdpi/ic_launcher.png b/res/mipmap-xhdpi/ic_launcher.png
similarity index 100%
rename from res/drawable-xhdpi/ic_launcher.png
rename to res/mipmap-xhdpi/ic_launcher.png
diff --git a/res/drawable-xxhdpi/ic_launcher.png b/res/mipmap-xxhdpi/ic_launcher.png
similarity index 100%
rename from res/drawable-xxhdpi/ic_launcher.png
rename to res/mipmap-xxhdpi/ic_launcher.png
diff --git a/res/drawable-xxxhdpi/ic_launcher.png b/res/mipmap-xxxhdpi/ic_launcher.png
similarity index 100%
rename from res/drawable-xxxhdpi/ic_launcher.png
rename to res/mipmap-xxxhdpi/ic_launcher.png
-- 
2.17.1

